src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,38 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+
+type TabsType = {
+  id: string;
+  title: string;
+  content: string;
+};
+
+export const Tabs = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}: {
+  tabs: TabsType[];
+  selectedTabId: string;
+  onTabSelected: React.Dispatch<React.SetStateAction<string>>;
+}) => {
```

Comment:
Move it to props type, please 
```suggestion
}: Props) => {
```

or you can use React.FC
```
export const Tabs: React.FC<Props> = ...